### ISSUE TITLE
HTML escaping in documentation

### ISSUE THREAD

**Issue by abudden** [nim-lang/Nim#4373]
I was reading the filters documentation (http://nim-lang.org/docs/filters.html) and was rather confused by the description of the `conc` option to `stdtmpl`.  After a bit of investigation (and looking at the html source), I realised that the default value is `" & "`, not `"   "` as described - this made it much clearer!

The html generated is (literally) `" & "` whereas it should be `" &amp; "`.

**Comment by Araq** [comment]
That's not true, the HTML looks like:

```
<dt><tt class="docutils literal"><span class="pre">conc: string = &quot; &amp; &quot;</span></tt></dt>
<dd>the operation for concatenation</dd>
```

And renders perfectly fine in the browser.

**Comment by abudden** [comment]
@Araq.  Hmm... that's odd.  I guess this is something going odd with Chrome.  I've checked the source again and it indeed looks correct.  I'm not sure where I looked to see the incorrect html [edit: actually, it might have been the "Inspect" option, which shows the developer tools and seems to expand the `&amp;`, so it looks like that was me jumping to invalid conclusions].

Here's a screenshot from Chrome Version 51.0.2704.103 m on Windows 7:

![chrome](https://cloud.githubusercontent.com/assets/670607/16761003/2f0bcd48-4816-11e6-9071-2cdf58f84daf.png)

However, it looks okay in internet explorer and firefox (the following is internet explorer):

![iexplore](https://cloud.githubusercontent.com/assets/670607/16761057/70807de6-4816-11e6-90da-2cd50d9a00d4.png)

I guess it's something odd about Google Chrome.  Weird...
